static/js/util/profile_edit.js

```diff
@@ -334,10 +340,10 @@ export function boundMonthYearField(keySet, label) {
  * pass an option callback if you need nested validation
  * (see EmploymentTab for an example)
  *
- * @param next {String} URL to redirect to after successful validation
+ * @param afterSave {String} URL to redirect to or function to call after successful validation 
  * @param nestedValidationCallback {func} If present, a function to retrieve validation fields
  */
-export function saveAndContinue(next, nestedValidationCallback) {
+export function saveAndContinue(afterSave, nestedValidationCallback) {
```

Comment:
I think we can rewrite this to return a `Promise` instead, the one returned by `saveProfile.then()` below. Then the calling function could do `this.context.router.push('')` or execute other custom code. What do you think?